android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -358,16 +358,22 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 return true;
 
             case R.id.action_share:
+                StringBuilder speakersNameBuilder  = new StringBuilder();
+                for(Speaker speaker : session.getSpeakers()){
+                    speakersNameBuilder.append(speaker.getName()+", ");
+                }
+                speakersNameBuilder.delete(speakers.length()-3,speakers.length()-1);
```

Comment:
What is the use of this line ? 
The build fails because it can't find the ```speakers``` variable. Is it even working locally ?
Please always run the application after you make changes before pushing.

Reply:
Also add appropriate spacing after for, +, - and comma